python/feathub/feature_tables/sinks/kafka_sink.py

```diff
@@ -38,17 +40,26 @@ def __init__(
                              etc.
         :param producer_properties: Optional. If it is not None, it contains the extra
                                     kafka producer properties.
+        :param key_format_config: Optional. The configuration of the format for Kafka
+                                  message key.
+        :param value_format_config: Optional. The configuration of the format for Kafka
+                                    message value.
         """
         super().__init__(
             name="",
             system_name="kafka",
             properties={"bootstrap_server": bootstrap_server, "topic": topic},
             data_format=value_format,
+            format_config=value_format_config,
```

Comment:
The `Sink#__init__(...)` does not have this named parameter.

And it might be simpler to encode extra configs in `properties` in a pattern similar to Kafka Source [1].

[1] https://github.com/alibaba/feathub/blob/master/python/feathub/processors/flink/table_builder/tests/test_kafka_source_sink.py#L116